install/helm/choreo/crds/configurationgroup-crd.yaml

```diff
@@ -0,0 +1,204 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    cert-manager.io/inject-ca-from: choreo-system/choreo-serving-cert
+    controller-gen.kubebuilder.io/version: v0.16.4
+  name: configurationgroups.core.choreo.dev
+spec:
+  group: core.choreo.dev
+  names:
+    categories:
+    - choreo
+    - all
+    kind: ConfigurationGroup
+    listKind: ConfigurationGroupList
+    plural: configurationgroups
+    shortNames:
+    - configgrp
+    singular: configurationgroup
+  scope: Namespaced
+  versions:
+  - additionalPrinterColumns:
+    - jsonPath: .metadata.annotations.core\.choreo\.dev/display-name
+      name: DisplayName
+      type: string
+    - jsonPath: .metadata.labels.core\.choreo\.dev/organization
+      name: Organization
+      type: string
```

Comment:
Do we need this additional column for kubectl? Our other resources don't have this.